java/opendcs/src/main/java/decodes/db/LocationLevelSpec.java

```diff
@@ -0,0 +1,94 @@
+package decodes.db;
+
+import decodes.sql.DbKey;
+import opendcs.dao.CachableDbObject;
+
+import java.util.Date;
+
+/**
+ * Inner class representing a Location Level Specification
+ * Implements CachableDbObject for caching support
+ */
+public class LocationLevelSpec implements CachableDbObject
```

Comment:
Don't put new code in decodes/db, put it in org/opendcs.

Also, unless we want to figure out a proper "OpenDCS" "Location Meta Data" these should be under a /cwms names to indicate they aren't generic.